fusor-ember-cli/app/controllers/review/installation.js

```diff
@@ -242,33 +242,18 @@ export default Ember.Controller.extend(NeedsDeploymentMixin, OpenshiftMixin, {
     return this.get('storageSize') + ' GB';
   }),
 
-  deploymentButtonAction: Ember.computed('hasSubscriptionsToAttach', function() {
-    if (this.get('showWarningMessage')) {
-        return "showContinueDeployModal";
-    } else if (this.get('hasSubscriptionsToAttach')) {
-        return "attachSubscriptions";
-        return "installDeployment";
-    } else if (this.get('showWarningMessage')) {
-        return "showContinueDeployModal";
-    }
-  }),
-
   closeContinueDeployModal() {
-    this.set('continueDeploymentModalOpen', false);
-    this.set('continueDeploymentModalClosed', true);
-    this.set('modalOpen', false);
+    this.set('openModal', false);
   },
 
   actions: {
     showContinueDeployModal() {
       this.set('deploymentInModal', this.get('model'));
```

Comment:
What error was thrown? I only see `deploymentInModal` in use for deleting deployments on the deployments route. I this necessary?

```
# grep deploymentInModal . -r
./components/tr-deployment.js:      this.get('targetObject').set('deploymentInModal', item);
./controllers/review/installation.js:      this.set('deploymentInModal', this.get('model'));
./controllers/deployment.js:  deploymentInModal: Ember.computed.alias("deploymentsController.deploymentInModal"),
./templates/deployments.hbs:                          deployment=deploymentInModal
```